packages/telemetry/src/index.js

```diff
@@ -20,6 +20,7 @@ export * from './make-slog-sender.js';
  * @typedef {MakeSlogSenderCommonOptions & Record<string, unknown>} MakeSlogSenderOptions
  * @typedef {object} MakeSlogSenderCommonOptions
  * @property {Record<string, string | undefined>} [env]
+ * @property {Pick<Console, 'debug' | 'log' | 'info' | 'warn' | 'error'>} [console]
```

Comment:
cf. [#10925 js-utils.js](https://github.com/Agoric/agoric-sdk/pull/10925/files#diff-324478d65037c66bbaeb4f33f060c89e227992aca8adb1461b6ad0eecc11939f):
```js
export const logLevels = /** @type {const} */ ([
  'debug',
  'log',
  'info',
  'warn',
  'error',
]);
Object.freeze(logLevels);

/** @typedef {(typeof logLevels)[keyof logLevels & number]} LogLevel */

/** @typedef {Pick<Console, LogLevel>} LimitedConsole */
```

```suggestion
 * @property {LimitedConsole} [console]
```

Reply:
Thanks!